features/option.feature

```diff
@@ -248,11 +272,11 @@ Feature: Manage WordPress options
       """
     And the return code should be 0
 
-    When I try `wp option list --search='auto_opt' --autoload=no`
+    When I try `wp option list --search='auto_opt' --autoload=nope`
     Then STDOUT should be empty
-    And STDERR should be:
+    And STDERR should contain:
```

Comment:
I changed this to contain because the error message has been updated (to include yes and no as valid options) and it didn't seem worth an entire extra Scenario for wp 6.6

I can add one if wanted

Reply:
Sounds fine to me 👍